chapters/01-spatial-data.qmd

```diff
@@ -21,3 +29,118 @@ using GeoMakie
 
 f, a, p = poly(df.geom)
 ```
+
+
+### Raster from scratch {#sec-raster-from-scratch}
+
+In this section, we are going to demonstrate the creation of rasters from scratch.
+We will construct two small rasters, `elev` and `grain`, which we will use in examples later in the book.
+Unlike creating a vector layer (see @sec-vector-layer-from-scratch), creating a raster from scratch is rarely needed in practice because aligning a raster with the proper spatial extent is challenging to do programmatically ("georeferencing" tools in GIS software are a better fit for the job).
+Nevertheless, the examples will be helpful to become more familiar with the **Rasters.jl** data structures.
+
+```{julia}
+using Rasters
+import GeoFormatTypes as GFT
+```
+
+Conceptually, a raster is an array combined with georeferencing information, whereas the latter comprises:
+
+-   A transformation matrix, containing the origin and resolution, thus linking pixel indices with coordinates in a particular coordinate system
+-   A CRS definition, specifying the association of that coordinate system with the surface of the earth (optional)
```

Comment:
CRS acronym is not introduced. Let's not make the crs optional? Otherwise the transformation is useless, so either define it because it's in the real world, or use the engineering crs.

Reply:
Agreed.  I've also reformatted the point above this to talk about "lookup vectors", but I'm not sure my explanation was the best there.  A lot of this can probably also be pinched to go into DD/Rasters docs...